qa/L0_java_resnet/test.sh

```diff
@@ -40,14 +40,21 @@ fi
 
 # Models
 DATADIR=/data/inferenceserver/${REPO_VERSION}
+MODEL_REPO=`pwd`/models
 
 # Create local model repository
-mkdir -p models/
-cp -r $DATADIR/perf_model_store/resnet50* models/
+mkdir -p ${MODEL_REPO}
+# TODO: fix build to support GPU only resnet50v1.5_fp16_savedmodel
+for BACKEND in _fp32_libtorch _fp32_onnx; do
+    cp -r $DATADIR/perf_model_store/resnet50${BACKEND} ${MODEL_REPO}/
+    echo ${MODEL_REPO}/resnet50${BACKEND}/config.pbtxt
+    sed -i "s/kind: KIND_GPU/kind: KIND_CPU/" ${MODEL_REPO}/resnet50${BACKEND}/config.pbtxt
```

Comment:
I thought the models can be deployed on GPU? This suggests that is not the case? If it can and you want the test can also be run on cpu-only environment, you can change it to KIND_AUTO and Triton should assign the proper instance kind for you